### ISSUE TITLE
Question about label fraction in the code

### ISSUE THREAD

**Issue by Natguo** [rookiehb/DisMAE#4]
Thank you for sharing your source code. I’ve reimplemented the work and obtained the same results as reported in the paper. However, I’m a bit confused about the label fraction in the code. It seems that only the holdout_fraction parameter affects the fraction of labeled data used in training—is that correct?

**Comment by Natguo** [comment]
If I only change the val_fraction, setting it to 0.1 or 1.0 gives the same results as val_fraction = 1.0 in the paper. However, when I set holdout_fraction = 0.9 (which, as I understand it, means the fraction of labeled training data is 1 - holdout_fraction), the average accuracy is around 15, compared to 20.3 in the paper.

**Comment by rookiehb** [comment]
Thanks for your interest. Yes, but you should also change other parameters to fit into experiments with different setups.
